lualatex-math.dtx

```diff
@@ -326,10 +326,10 @@
 %    \begin{macrocode}
 \@ifl@t@r \fmtversion { 2020/10/01 } {
   \cs_new_protected_nopar:Npn \@@_before_package:nn #1 #2 {
-    \AddToHook { package/before/#1 } { #2 }
+    \AddToHook { package/#1/before } { #2 }
```

Comment:
Can you make these conditional on the format version that introduced these new hooks? Otherwise I guess the package won't work with old but still supported format versions.

Reply:
Done. (Should have seen that this is considered here.) Though I wonder how much backward compatibility has to be ensured here; surely people update the kernel if they update this package?

Reply:
As a side note, there is `\IfFormatAtLeastTF` in the kernel since `2020/10/01`. Once this format version is required, it may be worth switching to this command rather than using `\@ifl@t@r \fmtversion`.